### PR TITLE
update API to ES5 style

### PR DESCRIPTION
**Pull request by k-toumura**
Change API for ES5 style for browser compatibility.

### CODE REVIEW

**Comment by knolleary** [comment]
It would be helpful if the design note included some examples of the api in use.

It includes a number of use cases, but it doesn't go on to show how the api would be used to address those use cases.

For example, if I have a String contain flow json, what do I do to use this api?

This is still marked as a Draft PR - aside from my comment above, are there more changes you plan to make as part of this PR?

File: designs/flow-manipulation-api/README.md [hunk 1]
```diff
@@ -122,8 +122,9 @@ Based on [Design: Flow file format v2](https://github.com/node-red/node-red/wiki
 - static parseFlow(string or array)
   - create Flowset from flow.json string or parsed array
   - returns: FlowSet object
-- generator()
-  - iterate over all nodes in this flowset.
+- getAllNodesArray()
```
Comment (knolleary) [inline designs/flow-manipulation-api/README.md]: `getNodes` would be a more consistent name for this method. Same comment across the other classes that include this function name.

Comment (k-toumura) [reply designs/flow-manipulation-api/README.md]: I agree.  I updated the PR for this.

**Comment by k-toumura** [comment]
For consistence with the name of PR, I update only for API style matter.
In next PR, I'll add examples for API usage, as @knolleary commented above.

**Comment by knolleary** [comment]
I think this design has been superseded by https://github.com/node-red/flow-parser

Any further discussion on the API should happen in that repo.

**Pull request closed without merging**